README.md

```diff
@@ -83,7 +81,108 @@ make localnet-start
 make localnet-stop
 ```
 
+# Finschia Mainnet/Testnet
+
+## Public endpoints
+
+| Chain            | Chain ID   | Endpoint                                                                                                                                                                          |
+| ---------------- | ---------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Finschia mainnet | finschia-2 | https://finschia-api.finschia.io (REST)<br>https://finschia-rpc.finschia.io/ (RPC)<br>finschia-grpc.finschia.io:443 (gRPC)<br>https://finschia-api.finschia.io/swagger/ (Swagger) |
+| Ebony testnet    | ebony-2    | https://ebony-api.finschia.io (REST)<br>https://ebony-rpc.finschia.io (RPC)<br>ebony-grpc.finschia.io:443 (gRPC)<br>https://ebony-api.finschia.io/swagger/  (Swagger)             |
+
+## Genesis/Snapshot files for Finschia mainnet/testnet
+
+* Genesis file
+    * Finschia mainnet
+        * [finschia-genesis.tgz](https://vos.line-scdn.net/finschia-2-fileshare/datafile/finschia-prod-2/finschia-2-genesis.tgz)
+    * Ebony testnet
+        * [ebony-genesis.tgz](https://vos.line-scdn.net/finschia-2-fileshare/ebony-prod-2/genesis-file.tgz)
+* Snapshot file
+    * Finschia mainnet (pruned)
+        * [finschia-2-pruned.tgz](https://finschia-quicksync.line-scdn.net/finschia-2/pruned/finschia-2-pruned.tgz)
+    * Ebony testnet (pruned)
+        * [ebony-2-pruned.tgz](https://finschia-quicksync.line-scdn.net/ebony-2/pruned/ebony-2-pruned.tgz)
```

Comment:
How about adding the data size if possible?

Reply:
Of course, It's possible.
But it's little bit redundant, web browsers show us the size and progress when we click the download link.

Reply:
And size information is static, that could be outdated when we update the file.